packages/fabric8-ui/src/app/dashboard-widgets/space-settings-widget/space-settings-widget.component.ts

```diff
@@ -0,0 +1,187 @@
+import { Component, Input, OnInit, ViewEncapsulation, ViewChild } from '@angular/core';
+import { Broadcaster } from 'ngx-base';
+import { ModalDirective } from 'ngx-bootstrap';
+import { uniqBy } from 'lodash';
+import { CollaboratorService, Space, AreaService, SpaceService, Spaces } from 'ngx-fabric8-wit';
+import { of as observableOf, BehaviorSubject, Subscription, Observable, empty } from 'rxjs';
+import { switchMap, tap, first, catchError, concatMap } from 'rxjs/operators';
+import { User } from 'ngx-login-client';
+import { SpaceNamespaceService } from '../../shared/runtime-console/space-namespace.service';
+
+@Component({
+  encapsulation: ViewEncapsulation.None,
+  selector: 'fabric8-space-settings-widget',
+  templateUrl: './space-settings-widget.component.html',
+  styleUrls: ['./space-settings-widget.component.less'],
+})
+export class SpaceSettingsWidgetComponent implements OnInit {
+  @Input() userOwnsSpace: boolean;
+  @Input() userIsSpaceAdmin: boolean;
+  @Input() user: User;
+  @Input() space: Space;
+  @ViewChild('spaceDescription') description: any;
```

Comment:
Isn't description should be string type?

Reply:
It is HTMLref type, not string.